### PR TITLE
Update roadmap and readme

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

File: .github/ROADMAP.md [hunk 1]
```diff
@@ -1,37 +1,101 @@
...
+This will largely be handled by [divyashreepathihalli](https://github.com/divyashreepathihalli), but there
+may be the opportunity for community contributors to contribute support in specific layers after a reference example is available.
+
+### Classification Model Training Template
+
+KerasCV will offer a state of the art training template for produce high quality, efficient
+CNN classifiers.  The template will be written to train on the `imagenet2012` dataset,
+but will support custom dataset usage.  This template will support recovery from failure,
+checkpointing, metric logging to TensorBoard, and more.  The template will not require
+users to "nurse" the job.  [lukewood](https://github.com/lukewood) will be kicking off
```
Comment (LukeWood) [inline .github/ROADMAP.md:43]: Note: update to mention Scott

File: .github/ROADMAP.md [hunk 2]
```diff
@@ -1,37 +1,101 @@
...
+Same as Classification template but for bounding box detection
+
+### Visualization Tools
+
+The new `visualization` directory will contain methods to visualize various components of
+the training pipeline: i.e. visualize bounding boxes, segmentation maps, and more!
+
+The specific feature set here is still being sorted out.
+
+### Explainability Tools
```
Comment (LukeWood) [inline .github/ROADMAP.md:60]: Maybe I can design this

File: README.md [hunk 3]
```diff
@@ -7,16 +7,18 @@
...
-the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that 
+KerasCV can be understood as a horizontal extension of the Keras API: the components are new first-party
+Keras objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive
+the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that
 are maintained by the Keras team itself (unlike TFAddons).
 
-Currently, KerasCV is operating pre-release.  Upon launch of KerasCV 1.0, full API docs and code examples
-will be available.  To learn more about the project, please check the [roadmap](.github/ROADMAP.md).
+KerasCV's primary goal is to provide a coherent, elegant, and pleasant API to train state of the art computer vision models.
+Users should be able to train state of teh art models using only `Keras`, `KerasCV`, and TensorFlow core (i.e. `tf.data`) components.
```
Comment (LukeWood) [inline README.md:19]: the*

File: .github/ROADMAP.md [hunk 4]
```diff
@@ -1,37 +1,101 @@
...
 ## Non-Goals
 - **KerasCV is not a repository of blackbox end-to-end solutions (like TF Hub or ModelGarden).**
 
-    For the time being, it is focused on modular and reusable building blocks. We expect models in 
+    For the time being, it is focused on modular and reusable building blocks. We expect models in
     ModelGarden or TF Hub to eventually be built on top of the building blocks provided by KerasCV.
 
-    In the process of developing these building blocks, we will by necessity implement end-to-end 
-    workflows, but they're intended purely for demonstration and grounding purposes, they're not 
+    In the process of developing these building blocks, we will by necessity implement end-to-end
```
[the remark this answers is not in the document]
Comment (LukeWood) [reply .github/ROADMAP.md:72]: For now they will live inside of the code examples/demos.

[the remark this answers is not in the document]
Comment (bhack) [reply .github/ROADMAP.md:72]: Ok so they are not exported in the namespace and they are not part of the API.

**Review by divyashreepathihalli — APPROVED** [review]

**Pull request merged**